python/desc/monitor/dbConnection.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from lsst.sims.catalogs.generation.db import DBObject
+
+class dbInterface(object):
+
+    def __init__(self):
```

Comment:
That is in fact the mechanism we were using planning on explicitly in the code with default arguments to those parameters. I did not notice that this was already taken care of. I don't worry much about the hostname, though this can be similarly removed.